springwolf-plugins/springwolf-cloud-stream-plugin/src/test/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/cloudstream/FunctionalChannelBeanDataTest.java

```diff
@@ -89,7 +89,8 @@ void testConsumerBeanWithGenericPayload() throws NoSuchMethodException {
         Set<FunctionalChannelBeanData> data = FunctionalChannelBeanData.fromMethodBean(method);
 
         assertThat(data)
```

Comment:
This is not a batch listener - it is intended that the entire payload here is a List.
Please create a new bean method where the Generic type of the `Consumer` is `Message<String>` and add a specific test for this case.